config.yml

```diff
@@ -39,14 +39,8 @@ desktop:
   asset_impact_data_path: "./ai_inputs"
   factset_data_path: "./factset_inputs"
 
-docker:
-  data_prep_outputs_path: "/mnt/outputs"
-  asset_impact_data_path: "/inputs"
-  factset_data_path: "/inputs"
-
 
 2021Q4:
-  inherits: docker
```

Comment:
let's leave these in, otherwise the configs that don't have a matching `202xQ4_docker` are broken